packages/graphback-runtime-mongodb/tests/MongoDataProviderTest.ts

```diff
@@ -279,3 +222,122 @@ describe('MongoDBDataProvider Basic CRUD', () => {
     expect(next.createdAt).toEqual(createDate.getTime())
   })
 });
+
+describe('MongoDB indexing', () => {
```

Comment:
When/if we add a file for indexing, the tests can be moved too.

Reply:
What do you think about letting these tests stay as a sort of integration test, if yes, should I do some unit tests for this as well?